src/components/checkbox/checkbox.js

```diff
@@ -26,4 +26,34 @@ function initCheckbox(doc = document) {
   };
 }
 
+
+  //sets indeterminate state 
+  const intdeterminateCheckbox = [...document.querySelectorAll('[aria-checked=mixed]')];
```

Comment:
Should we move the block below here to the `init` method?.

Also:

`intdeterminateCheckbox` -> `indeterminateCheckbox`.

Should we just use `Array.from` on the result of `querySelectorAll`?

Reply:
Probably a stupid question but don't `Array.from `and` [... ]` do the same thing?

Reply:
The usage should totally be fine in newer browsers, but figured it might be safer to use `Array.from` since some browsers don't treat the result type of `querySelectorAll` as an iterable, so the spread operator couldn't be applied to it.

Reply:
@joshblack Good point - Babel turns any array-likes to arrays with `…` operator and that’s why our widespread usage of `…` operator works with IE11, but I guess we’d better change the practice to be more spec-complient.